app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class UserMailer < ApplicationMailer
+
+  default from: 'ankitadixit.rails@gmail.com'
+
+  def welcome(user)
+    @user = user
+    @greeting = 'Helllo !!!'
```

Comment:
No need of this instance variable.
Directly use `Helllo !!!` in `welcome.html.erb`